src/app/2025/(home)/_components/Banner.tsx

```diff
@@ -60,18 +60,20 @@ export const Banner = ({
             <fieldset className="w-fit rounded-md border border-gray-500/50">
               <legend>WHO ARE YOU SIGNING UP AS?</legend>
               <div className="flex flex-row items-center justify-start gap-4 p-4">
-                <Link
-                  className="button-white bg-white text-[.7rem] leading-snug text-black sm:text-[1rem]"
-                  href={UNSW_ONLY_SIGNUP_FORM_URL}
-                >
-                  I&apos;M A UNSW STUDENT
-                </Link>
-                <Link
-                  className="button-white bg-white text-[.7rem] leading-snug text-black sm:text-[1rem]"
-                  href={INTER_UNI_SIGNUP_FORM_URL}
-                >
-                  I&apos;M FROM ANOTHER UNI
-                </Link>
+                <div className="flex flex-1 gap-4 w-full">
+                  <Link
+                    className="flex flex-1 min-w-0 items-center justify-center rounded-md bg-white px-4 py-2 text-center text-[.7rem] leading-snug text-black sm:text-[1rem]"
```

Comment:
Nice use of flex box! However, let's still use the `.button-white` utility class - it already includes many of the same Tailwind classes. This helps keep our codebase consistent and makes it easier to manage and reuse button styles in the future 😉. e.g.
```suggestion
                    className="button-white items-center rounded-md text-[.7rem] sm:text-[1rem]"
```
-> then just update the [`.button-white`](https://github.com/UNSW-Robotics-and-Mechatronics-Society/ramsoc-sumobots-website/blob/fix/sign-up-buttons/src/app/styles.css#L143) class accordingly.

Also [optional style choice], I think text aligning to left looks better but up to you
![image](https://github.com/user-attachments/assets/0e97b8d7-6ff2-447a-b1a8-600dc56346b8)